plugins/restful/RestfulEntityBase.php

```diff
@@ -324,15 +355,17 @@ public function viewEntity($id) {
         }
 
         if (empty($info['formatter'])) {
-          if ($sub_wrapper instanceof EntityListWrapper) {
-            // Multiple values.
-            foreach ($sub_wrapper as $item_wrapper) {
-              $value[] = $this->getValueFromProperty($wrapper, $item_wrapper, $info, $public_field_name);
+          if (field_is_translatable($this->entityType, field_info_field($info['property']))) {
```

Comment:
Don't we need to check that 'property' is a Field API field before doing `field_info_field($info['property'])`.